be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -98,26 +107,21 @@ class ExchangeSinkOperator::Channel {
 
     ExchangeSinkOperator* _parent;
 
-    TUniqueId _fragment_instance_id;
-    PlanNodeId _dest_node_id;
-
-    std::unique_ptr<vectorized::Chunk> _chunk;
-    bool _is_first_chunk = true;
+    const TNetworkAddress _brpc_dest_addr;
+    const TUniqueId _fragment_instance_id;
+    const PlanNodeId _dest_node_id;
+    const int32_t _channel_id;
 
-    TNetworkAddress _brpc_dest_addr;
-
-    PTransmitChunkParamsPtr _chunk_request;
+    const bool _enable_exchange_pass_through;
+    PassThroughContext _pass_through_context;
 
+    bool _is_first_chunk = true;
     doris::PBackendService_Stub* _brpc_stub = nullptr;
-
-    size_t _current_request_bytes = 0;
-
+    std::vector<std::unique_ptr<vectorized::Chunk>> _chunks;
```

Comment:
Elements of _chunk and exchange source operator instances is one-to-one mapping,  so one instance has a corresponding chunk in this vector,  maybe a clear name would improve unstandablility of code. i.e. 
_chunk_per_dest_operator

Reply:
Here are two situations
1. with shuffle, size of `_chunks ` is equal with dest pipeline's dop
2. without shuffle, size of `_chunks`  is always 1, which is not equal with dest pipeline

is `logical_shuffle_chunks` better?